src/main/java/org/opensearch/security/support/ConfigHelper.java

```diff
@@ -57,6 +57,7 @@
 
 import static org.opensearch.core.xcontent.DeprecationHandler.THROW_UNSUPPORTED_OPERATION;
 
+@Deprecated
```

Comment:
Any reason not to delete?

Reply:
Yes so far I split up logic with the cluster state update and existing one. So in case of any problem we can switch back to old the logic.